logrotate.c

```diff
@@ -2600,6 +2600,22 @@ int main(int argc, const char **argv)
 		break;
 	case 'V':
 	    fprintf(stderr, "logrotate %s\n", VERSION);
+	    fprintf(stderr, "\n");
+	    fprintf(stderr, "    Default mail command: %s\n", DEFAULT_MAIL_COMMAND);
+	    fprintf(stderr, "    Compress command:     %s\n", COMPRESS_COMMAND);
+	    fprintf(stderr, "    Uncompress command:   %s\n", UNCOMPRESS_COMMAND);
+	    fprintf(stderr, "    Compress extension:   %s\n", COMPRESS_EXT);
+	    fprintf(stderr, "    State file path:      %s\n", STATEFILE);
```

Comment:
I think we should use the ```Default ``` prefix in all the lines above.  This is especially important for the state file, which can be overridden by a command-line option (and certain distros use the option in default cron jobs to override logrotate's default) but I would put it everywhere to stay consistent.